scripts/build.ps1

```diff
@@ -23,34 +23,36 @@ Push-Location (Join-Path $workpath "v8build\v8")
 # Generate the build system
 $gnargs = 'v8_enable_i18n_support=false is_component_build=false v8_monolithic=true v8_use_external_startup_data=false treat_warnings_as_errors=false'
 
-if ($Configuration -like "*android") {
-    $gnargs += ' use_goma=false target_os=\"android\" target_cpu=\"' + $Platform + '\"'
+# commenting this out for now since Android build is at android/build.sh
+
+# if ($Configuration -like "*android") {

```

Comment:
># if  [](http://example.com/codeflow?start=0&length=5)

(nit) should all this just be removed?

Reply:
I'll leave the scripts like this for now to match master.